docs/handbook/systems-administration.md

```diff
@@ -28,201 +28,354 @@ activities and many more are referred to as system administration.
+* **username** the name of the account, which is frequently displayed on the login
+  screen, depending on the login display manager in use.
+  The username format is briefly described in [passwd(4)](https://illumos.org/man/4/passwd)
+* **password** the password required to access the account.
+                  *Passwords are used for security concerns.
+                  Accounts without a password should are
+                  a security hazard.*
 * **comment** (also referred to as gecos) - account description
-Often this filed is set to real user name.
-* **home directory** - a path where user will be after he logs into the system.
-* **login shell** - user's initial shell program.
-* **password last change time** - time when the password was lastly changed.
+This is often the user's real name.
+* **home directory** the path to the directory which is _home_ to the user and
+  where the user is placed by default after logging onto the system.
+* **login shell** user's initial shell program.
+* **password last change time** time at which the the user password was last changed.
 * password aging information - several [shadow(4)](https://illumos.org/man/4/shadow) fields determining when password should be changed
-    * **min** - the minimum number of days required between password changes;
-    * **max** - the maximum number of days the password is valid;
-    * **warn** - the number of days before password expires that the user is warned.
+    * **min** the minimum number of days required between password changes;
+    * **max** the maximum number of days the password is valid;
+    * **warn** the number of days before password expires that the user is warned.
+
+### [System (or Service) accounts](#accounts-service)
+
+System accounts, including the root account, are designed to be used for
+administrative purposes. While all such tasks can be performed using the root
+account, system accounts all have limited powers with respect to root, thereby
+delegating only those powers necessary to carry out an administrative task to a
+particular system account. System accounts often supply a specific function such
+as managing mail accounts or services to the network such as DNS, SMTP or WWW.
+
+All system accounts have a `UID` less than 100.
+
+All system accounts -- the one notable exception being root -- are supplied
+with either `LK` or `NP` in the second column of the `/etc/shadow` file. In other
+words, it is possible to obtain a list of all system accounts:
 
-### Service accounts
+```bash
+# more /etc/shadow | grep -e NP -e LK | cut -d: -f1
+```
+
+  | **Service account**  | **Purpose**|
+  |----------------------|----------|
+  | `nobody`             | for services that require no privileges |
+  | `pkg5srv`            | for the `pkg(5)` server |
+  | `webservd`           | for web servers |
+  |  ...                 |                 |
+
+
+#### [Superuser account](#accounts-superuser)
```

Comment:
It's not possible to define a link in this way. This section can be linked to in mkdocs using the name of the heading - `e.g. [example link](#superuser-account)`. It's not possible to make use of a different name.